login-form/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>login page</title>
+    <link rel="stylesheet" href="style.css" />
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="container flex">
+      <div class="login-form flex">
+        <div class="element">
+          <div>username</div>
+          <div><input type="email" class="username" /></div>
+          <div class="msgArea uname"></div>
+        </div>
+        <div class="element">
+          password
+          <div><input type="password" class="password" /></div>
+
+          <div class="msgArea pwd"></div>
+        </div>
+        <div class="element">
+          <button class="login-btn" onclick="validateFieldsNotEmpty()">
```

Comment:
onclick inside of html isn't something we should practice. Handling events in JS is a better way.

Reply:
@ksrscharan

Reply:
Implemented the same technique. just check code @Navaneeth-84 .

Reply:
so silly of me to place script into the `head` tag. got it 😁